calico/reference/kube-controllers/prometheus.md

```diff
@@ -20,7 +20,7 @@ existing metrics.
 | ------------- | --------------- |
 | `ipam_blocks_per_node` | Number of IPAM blocks, indexed by the node to which they have affinity. |
 | `ipam_allocations_per_node` | Number of Calico IP allocations, indexed by node on which the allocation was made. |
-| `ipam_borrowed_allocations_per_node` | Number of Calico IP allocations borrowed from a non-affine block, indexed by node on which the allocation was made. |
+| `ipam_allocations_borrowed_per_node` | Number of Calico IP allocations borrowed from a non-affine block, indexed by node on which the allocation was made. |
```

Comment:
Yes, agreed, and yet there they are in the code. I think I'm missing something. Perhaps this metric only appears in certain scenarios?

I decided to submit the PR anyway on the basis of the mismatch in the code/docs, which in any case cannot be correct, but I should probably understand why the grep doesn't show it.

Reply:
I _think_ that's because the metric won't be exported unless there are borrowed allocations in the cluster. Otherwise prometheus won't know about it.